packages/miniflare/src/plugins/shared/index.ts

```diff
@@ -191,10 +193,15 @@ export function getPersistPath(
 	// keep Miniflare 2's behaviour, so persist to a temporary path which we
 	// destroy on `dispose()`.
 	const memoryishPath = path.join(tmpPath, pluginName);
-	if (persist === undefined || persist === false) {
+	if (persist === false) {
 		return memoryishPath;
 	}
 
+	// If `persist` is undefined, use either the default path or fallback to the tmpPath
+	if (persist === undefined) {
+		return path.join(defaultPersistRoot ?? tmpPath, pluginName);
+	}
```

Comment:
So I think you are changing the default for Miniflare, right?
Now if you don't provide a persist path then instead of it being in-memory it writes to the default persist path.

Reply:
Before this change, when `persist` is undefined, it will always be the in-memory path which is based on the `tmpPath`. So this changes the behavior only if `defaultPersistRoot` is set.